src/corelibs/U2View/src/ov_msa/MSAEditor.h

```diff
@@ -150,31 +150,38 @@ protected slots:
     void alignSequencesFromFilesToAlignment();
 
 public:
-    QAction *buildTreeAction;
-    QAction *alignAction;
-    QAction *alignSequencesToAlignmentAction;
-    QAction *realignSomeSequenceAction;
-    QAction *setAsReferenceSequenceAction;
-    QAction *unsetReferenceSequenceAction;
-    QAction *gotoAction;
-    QAction *searchInSequencesAction;
-    QAction *searchInSequenceNamesAction;
-    QAction *openCustomSettingsAction;
-    QAction *sortByNameAscendingAction;
-    QAction *sortByNameDescendingAction;
-    QAction *sortByLengthAscendingAction;
-    QAction *sortByLengthDescendingAction;
-    QAction *sortByLeadingGapAscendingAction;
-    QAction *sortByLeadingGapDescendingAction;
-
-    QAction *convertDnaToRnaAction;
-    QAction *convertRnaToDnaAction;
+    QAction *buildTreeAction = nullptr;
+    QAction *alignAction = nullptr;
+    QAction *alignSequencesToAlignmentAction = nullptr;
+    QAction *realignSomeSequenceAction = nullptr;
+    QAction *setAsReferenceSequenceAction = nullptr;
+    QAction *unsetReferenceSequenceAction = nullptr;
+    QAction *gotoAction = nullptr;
+    QAction *searchInSequencesAction = nullptr;
+    QAction *searchInSequenceNamesAction = nullptr;
+    QAction *openCustomSettingsAction = nullptr;
+    QAction *sortByNameAscendingAction = nullptr;
+    QAction *sortByNameDescendingAction = nullptr;
+    QAction *sortByLengthAscendingAction = nullptr;
+    QAction *sortByLengthDescendingAction = nullptr;
+    QAction *sortByLeadingGapAscendingAction = nullptr;
+    QAction *sortByLeadingGapDescendingAction = nullptr;
+
+    QAction *convertDnaToRnaAction = nullptr;
+    QAction *convertRnaToDnaAction = nullptr;
 
 private:
-    PairwiseAlignmentWidgetsSettings *pairwiseAlignmentWidgetsSettings;
+    PairwiseAlignmentWidgetsSettings *pairwiseAlignmentWidgetsSettings = nullptr;
     MSAEditorTreeManager treeManager;
 };
 
+/** Set of custom menu actions in MSA editor. */
+class U2VIEW_EXPORT MsaEditorMenuType {
```

Comment:
Inherit it from `GObjectViewMenuType` to show the fact they are connected

Reply:
Fixed